### ISSUE TITLE
Scrying implies cards are in opposite order

### ISSUE THREAD

**Issue by mehlon** [Cockatrice/Cockatrice#3921]
<b>System Information:</b>
  
Client Version: 2.7.2-custom(unknown) (unknown)

  
_______________________________________________________________________________________

  
When scrying multiple cards, the interface shows the cards overlapping each other. 
Which card corresponds to the topmost card of the deck? The obvious guess for a new player is that the card which appears topmost visually, is the card that is topmost in the deck.
![cockatrice_ui](https://user-images.githubusercontent.com/13736092/77202549-d82fe680-6aee-11ea-921a-75615d6c8828.png)
In this screenshot, one would think the Mountain is the top card of the deck. The card that's most prominently visible is probably the one you would draw next, right? But actually, the card Scorch Spitter is the next card you would draw. 

I suggest changing it so that the card which is on top of the deck visually appears on top:
![newcockatrice_ui](https://user-images.githubusercontent.com/13736092/77205309-e41ea700-6af4-11ea-841a-86220340595a.png)

**Comment by ebbit1q** [comment]
No, this is how three cards look like when you grab them all at once, it makes perfect sense to me.

**Comment by ZeldaZach** [comment]
I could see following arena's method of having indicators on the page ("Top of deck") next to the first card

**Comment by ZeldaZach** [comment]
After discussing this further, I'd like to see some kind of indicator showing what's the top and bottom, but overall I'm happy with how it functions as-is. This would be a purely cosmetic change.
